### ISSUE TITLE
Executing scheduled transaction throws INSUFFICIENT_TX_FEE

### ISSUE THREAD

**Issue by teacoat** [hashgraph/hedera-sdk-js#957]
### Description

Moved from 2.7.1 to 2.9.1 and scheduled transactions that work in 2.7.1 no longer work and throw INSUFFICIENT_TX_FEE, when there is for sure enough for a tx fee.

### Steps to reproduce

1) Create a scheduled transaction, in our case it was an atomic transaction swapping NFT's for hbar
2) try it in 2.7.1
3) try it in 2.9.1

### Additional context

![image](https://user-images.githubusercontent.com/3913019/154865891-1107e96c-928e-4286-8e81-680fbd2afea5.png)

### Hedera network

mainnet

### Version

2.9.1

**Comment by danielakhterov** [comment]
I feel like more context is warranted since we do have an integration test which is passing for `ScheduleCreateTransaction` https://github.com/hashgraph/hedera-sdk-js/blob/main/test/integration/ScheduleCreateIntegrationTest.js Although in the test we use `Transaction.schedule()` to create a scheduled transaction instead of using `ScheduleCreateTransaction` directly.

**Comment by teacoat** [comment]
I don't think I'm doing anything too crazy, but I am using ScheduleCreateTransaction - 
```
let sched_trans = await new ScheduleCreateTransaction()
            .setAdminKey(this.client.operatorPublicKey)
            .setPayerAccountId(AccountId.fromString(payer))
            .setScheduledTransaction(trans);
```
The transaction itself is a simple atomic transaction, containing hbar transfers and nft transfers.

**Comment by danielakhterov** [comment]
I'll take a closer look later today, but we'll likely need to get this prioritized into our next sprint (which starts today).

**Comment by danielakhterov** [comment]
I _believe_ I've fixed the default transaction fees in https://github.com/hashgraph/hedera-sdk-js/releases/tag/v2.11.0-beta.1 I ran our e2e test suite just now and 99% pass; we do not set a max transaction fee for our e2e tests. Can you verify using the latest beta release fixes your use case?

**Comment by teacoat** [comment]
@danielakhterov hmmm I took a quick look and it seems to be the same, I'll gather some details tomorrow

**Comment by teacoat** [comment]
However - i did realize I am not setting a max transaction fee on the client, is it best practice to do this?

**Comment by danielakhterov** [comment]
We don't set the max transaction fee on client for our e2e tests so I wouldn't say that is best practice. I am a bit confused as to why it's still not working for you though. Our test uses `TopicMessageSubmitTransaction`, but that shouldn't make a difference. Hmmm, how many transfers are you doing in your requests? Maybe the current max fee for scheduled transactions is simply not enough for your particular request?

**Comment by teacoat** [comment]
I am just doing a 1hbar transfer from one account to another, I'll run through it again tonight

**Comment by teacoat** [comment]
@danielakhterov yeah i just tried again, same issue.

![image](https://user-images.githubusercontent.com/3913019/158272039-690637cb-2df8-4b5d-b09b-6177a222a903.png)

**Comment by teacoat** [comment]
All other transactions we have seem to work alright, just that scheduled transaction not working

**Comment by teacoat** [comment]
I tried putting a maxTransactionFee on the underlying transaction, that seems to have fixed it

**Comment by bugbytesinc** [comment]
When we upgraded the *Hedera Token Service Distribution Tool* to the latest SDK version we experienced the same issue.

**Comment by bugbytesinc** [comment]
![image (2)](https://user-images.githubusercontent.com/4412843/158889211-70bec01d-d7ca-45f4-a530-010181d7ea08.png)

After some spelunking, it appears the problem manifests due to the assumption that `freezeWith` will always be called on a `Transaction` object, however when the `Transaction` is enveloped inside a `ScheduledTransaction` the method does not get called, and therefore the default transaction max fee does not get set....looks like this results in the value of 0 being passed.  There appears to be no pre-checks on validating the interior transaction on this property on the server, so a schedule and sign will succeed without issue.  However, when the network receives enough signatures, it can't execute the transaction because the max fee is zero.

Explicitly setting the fee on the interior transaction object seems to work-around this issue, but for diversity and distributed interoperable ecosystem concerns, it would be very much better for our application to rely on the defaults.

**Comment by danielakhterov** [comment]
Oh wow, that's an excellent catch, thank you for looking into it! _I think_ I fixed this in 54867e49b441ee7e7dcc521307af5edadf9f75c9 and released https://github.com/hashgraph/hedera-sdk-js/releases/tag/v2.11.3
